app/assets/javascripts/footer_modal.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(() => {
```

Comment:
Ah ok, changed this. Back and forth navigation is a bit tricky, but this was the case beforehand as well. At least, the `window.history.pushState()` method will set an entry in the browser's history for the `#sponsors` slug and calling this link again will open the modal. While back navigation works most of the time after having opened the modal, forward navigation is not working properly. However, I don't think this is a problem, as we deal with a modal that a user can always easily close by clicking anywhere next to the modal or by clicking the "x" button on the bottom of the modal. And closing the modal flawlessly removes the `#sponsors` slug all the time. The modal can be easily opened again by clicking the "sponsors" links or by just visiting the link in the browser history.

Reply:
In my browser (Chrome), back navigation is broken after opening and closing a modal (also, the modal does not reappear). I would suggest we keep it simple and leave the modal out of the history. I think this would mean that footer_modal.js can be completely removed (at least in my development setting this worked fine). There are some suggestions how to fix this listed [here](https://github.com/turbolinks/turbolinks/issues/561), in particular [this](https://github.com/turbolinks/turbolinks/issues/219) seems relevant. But to me it seems not necessary to invest more time into this.

Reply:
True, probably not worth spending more time into this. I've removed the calls to `pushState()`. However, I still left the direct link `/#sponsors` in there, so that we can link to the modal directly if necessary (e.g. on a blog post or in a newspaper article).